app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitValidator.java

```diff
@@ -72,6 +74,15 @@ public boolean isHabitValid(String habitTitle,
             return false;
         }
 
+        if (!(privacyButtons.isHabitPrivate() || privacyButtons.isHabitPublic())){
+            _fragment.displayErrorMessage(HabitDialog.INCORRECT_PRIVACY);
+            return false;
+        }
+        else if (privacyButtons.isHabitPrivate() && privacyButtons.isHabitPublic()){
+            _fragment.displayErrorMessage(HabitDialog.INCORRECT_PRIVACY);
+            return false;
+        }
```

Comment:
Check whether no buttons are selected or both buttons are selected.

Reply:
Since both conditions lead to the **same outcome** (display the `INCORRECT_PRIVACY` error message), these should **either** be combined into one `if` block **or** display **different** error messages!

- [ ] fix `if` block